src/cosalib/container_manifest.py

```diff
@@ -25,35 +25,38 @@ def delete_local_container_manifest(repo, tag):
     runcmd(cmd)
 
 
-def push_container_manifest(repo, tag, digestfile, v2s2=False):
+def push_container_manifest(repo, main_tag, add_tags, digestfile, v2s2=False):
     '''
     Push manifest to registry
     @param repo str registry repository
-    @param tag str manifest tag
+    @param main_tag str manifest tag
+    @param add_tags list of additional tags to push
     @param digestfile str write container digest to file
     @param v2s2 boolean use to force v2s2 format
     '''
-    cmd = ["podman", "manifest", "push",
-           "--all", f"{repo}:{tag}", f"{repo}:{tag}"]
+    base_cmd = ["podman", "manifest", "push", "--all", f"{repo}:{main_tag}"]
     if v2s2:
         # `--remove-signatures -f v2s2` is a workaround for when you try
         # to create a manifest with 2 different mediaType. It seems to be
         # a Quay issue.
-        cmd.extend(["--remove-signatures", "-f", "v2s2"])
+        base_cmd.extend(["--remove-signatures", "-f", "v2s2"])
     if digestfile:
-        cmd.extend([f"--digestfile={digestfile}"])
-    runcmd(cmd)
+        base_cmd.extend([f"--digestfile={digestfile}"])
+    runcmd(base_cmd + [f"{repo}:{main_tag}"])
+    for tag in add_tags:
+        runcmd(base_cmd + [f"{repo}:{tag}"])
 
 
-def create_and_push_container_manifest(repo, tag, images, v2s2, digestfile):
+def create_and_push_container_manifest(repo, tags, images, v2s2, digestfile):
     '''
     Do it all! Create, Push, Cleanup
     @param repo str registry repository
-    @param tag str manifest tag
+    @param tags list of tags
     @param images list of image specifications (including transport)
     @param v2s2 boolean use to force v2s2 format
     @param digestfile str write container digest to file
     '''
-    create_local_container_manifest(repo, tag, images)
-    push_container_manifest(repo, tag, digestfile, v2s2)
-    delete_local_container_manifest(repo, tag)
+    main_tag = tags[0]
```

Comment:
feels like maybe we don't need to unpeel this here and we could just do it in `push_container_manifest()` directly? Would prevent adding a another argument to the function call. Not sure if there are other benefits to peeling it here.